library_schemas/score/hedwiki/HED_score_1.0.0.mediawiki

```diff
@@ -0,0 +1,1106 @@
+HED library="score" version="1.0.0"
+
+'''Prologue'''
```

Comment:
The prologue still seems to be an old prologue, rather than the new one with the references?

Reply:
There is more information in the epilogue!
Then both are shown in the browser viewer's 'Schema information' section.

Reply:
Ok, thanks!